pkg/limits/frontend/frontend.go

```diff
@@ -232,21 +212,29 @@ func (f *Frontend) ExceedsLimits(ctx context.Context, req *logproto.ExceedsLimit
 			}
 		}
 	}
+	f.metrics.tenantRejectedStreams.WithLabelValues(
+		req.Tenant,
+		ReasonExceedsMaxStreams,
+	).Add(float64(len(rejectedStreams)))
```

Comment:
It's OK to add `0` here I think, and avoids the need for another `if` statement.